### PR TITLE
first file

### PR DESCRIPTION
**Pull request by duolist**

### CODE REVIEW

File: DeliteliChisla.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+number = int(input("Введите число "))
+deliteli = [1]
```
Comment (Gregory664) [inline DeliteliChisla.py]: Whar's bad name?

File: DeliteliChisla.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+number = int(input("Введите число "))
+deliteli = [1]
+delitel = 2
+result = 0
+while delitel <= number // 2:
+    result = number % delitel
+    if result == 0:
+        deliteli.append(delitel)
```
Comment (Gregory664) [inline DeliteliChisla.py]: How it is work?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
 # python_learning
 python learning
+123
```
Comment (Gregory664) [inline README.md]: What is it "123"?

Comment (duolist) [reply README.md]: just test commit

File: DeliteliChisla.py (hunk 1 above)
Comment (duolist) [reply DeliteliChisla.py]: Ok. i'l rename it

Comment (duolist) [reply DeliteliChisla.py]: i fixed this trubles

**Pull request merged**